### PR TITLE
Improve UI and design

### PR DESCRIPTION
**Pull request by Uzay-G**
Dramatic improvements in UI and design of the page:

Sneak peek of the home page:

image

**Any** reviews / comments on the general changes and improvements are greatly appreciated if you clone the branch and check out the new design.

This PR design can / should be worked on to have an even better UI and feedback is extremely useful.

Todo:

- [x] - Add form for creation of sub dirs

### CODE REVIEW

**Comment by Uzay-G** [comment]
@srevinsaju if you have time this would be incredibly useful

File: archivy/templates/base.html [hunk 1]
```diff
@@ -38,190 +34,98 @@ <h3>
...
+				if (document.body.classList.contains("dark-theme"))
+				{
+				  theme = "light";
+				}
+				localStorage.setItem("theme", theme);
+				document.body.classList.toggle("dark-theme");
+			  });
+				
+			</script>
+		-->
```
Comment (srevinsaju) [inline archivy/templates/base.html:106]: remove commented code, if its no longer useful. No explanation of why it has been commented is given, perhaps we should just remove it

File: archivy/templates/click_web/command_form.html [hunk 2]
```diff
@@ -11,41 +11,25 @@ <h1 class="command-title">{{ command.name|title }}</h1>
...
+				<div class="pure-control-group">
+					{% if field.nargs == -1 %}
+						{{ macros.add_variadic_field_input(field) }}
+					{% else %}
+						<label for="{{ field.name }}">
+							{{ field.human_readable_name|capitalize }}
+						</label>
+						{% for arg in range(field.nargs) %}
+							{{ macros.add_field_input(field) }}
+						{% endfor %}
```
Comment (srevinsaju) [inline archivy/templates/click_web/command_form.html:25]: looks like you use `vim`. Please set tab = 2, for all `*.html` and `*.js`. and don't forget to lint them with `xmllint`

**Review by srevinsaju — CHANGES_REQUESTED** [review]

**Comment by srevinsaju** [comment]
Can we get a real tree here:
image

something like 
image

**Comment by Uzay-G** [comment]
Oh yeah that type of design is better, you're right.

**Comment by Uzay-G** [comment]
@srevinsaju wdyt of this?

image

**Comment by srevinsaju** [comment]
Looks Amazing :heart: !!

File: archivy/templates/click_web/command_form.html (hunk 2 above)
Comment (Uzay-G) [reply archivy/templates/click_web/command_form.html:25]: Is 2 spaces recommended for html?

Comment (srevinsaju) [reply archivy/templates/click_web/command_form.html:25]: 2 or 4, this one is 8. which is quite huge

**Comment by Uzay-G** [comment]
Alright I think we can merge this, @srevinsaju all good?

**Comment by srevinsaju** [comment]
This is still a draft PR, is work still pending?

**Comment by srevinsaju** [comment]
Also, the `bookmarklet` feature doesnt work, leads to 404

**Comment by Uzay-G** [comment]
What happens for you with the bookmarklet? If everything is up to date, it should work.

**Comment by Uzay-G** [comment]
Ok I see what you mean - I fixed it.

**Comment by Uzay-G** [comment]
Should be good now right?

**Comment by Uzay-G** [comment]
Alright I'm going to merge this in !

**Pull request merged**